contentcuration/contentcuration/static/js/administration/router.js

```diff
@@ -0,0 +1,293 @@
+var Backbone = require("backbone");
+var Models = require("edit_channel/models");
+var _ = require("underscore");
+
+// this is a function so that the defaults don't get overwritten
+const GET_DEFAULT_ROUTES = () => ({
+	'users': {name: 'users', page: 1, sortKey: "email", sortOrder: "ascending"},
+	'channels': {name: 'channels', page: 1, sortKey: "name", sortOrder: "ascending"},
+})
+
+const CHANNEL_FILTERS = {
+	all: {
+		label: "All",
+		queryParams: {}
+	}, 
+	live: {
+		label: "Live",
+		queryParams: { live: "True"},
+	},
+	published: {
+		label: "Published",
+		queryParams: { published: "True"},
+	},
+	public: {
+		label: "Public",
+		queryParams: { public: "True"},
+	},
+	private: {
+		label: "Private",
+		queryParams: { public: "False" },
+	},
+	can_edit: {
+		label: "My Channels",
+		queryParams: { editable: "True" },
+	},
+	staged: {
+		label: "Needs Review",
+		queryParams: { staged: "True" },
+	},
+	sushichef: {
+		label: "Sushi Chef",
+		queryParams: { ricecooker_version__isnull: "False" },
+	},
+	deleted: {
+		label: "Deleted",
+		queryParams: { deleted: "True" },
+	},
+}
+
+const CHANNEL_SORT_FILTERS = {
```

Comment:
`CHANNEL_FILTERS`, `CHANNEL_SORT_FILTERS`, `USER_FILTERS` and `USER_SORT_FILTERS` are all used to pass state into backbone views.